### PR TITLE
Addressing LSST Pub Board and SAC review comments on Ch 1 and 2

### LINKED ISSUE #611: Comments on Chapters 1, 2 and 12
Hello all,
 A PDF scan of my hand-written comments is attached.  This is the first posting from the SAC review of the whole white paper.  These chapters are an introduction (1), conclusion (12), and a discussion of the OpSim runs (2).  Apologies that this is not split into separate small issues.  
Overall, they are well-written and clear, and many of my comments are minor suggestions on wording, etc.  
 My more substantive concerns are as follows:
  -The text is inconsistent about whether the airmass limit of the main survey is 1.4 or 1.5.  
  -The basic parameters of the baseline cadence survey (declination limits, full definition of the mini-surveys, observing cadence in deep fields, etc., etc) are never explained. 
  -Most of Chapter 2 reads as it was written about 2 years ago.  minion_1016 is described as a “candidate” for the baseline cadence (it has since been officially adopted), and there are a lot of no-longer interesting references to the older baseline cadence.  Many of the problems described in this chapter are solved with V4 of OpSim, but that is never mentioned.  
   -Dithering is mentioned in passing multiple times in Chapter 2, but never properly explained. 
   -Chapter 12 needs a final forward-looking section, describing next steps: what experiments can now be done with V4, how complete we are with relevant metrics, etc.

[0781_001.pdf](https://github.com/LSSTScienceCollaborations/ObservingStrategy/files/924292/0781_001.pdf)

### LINKED ISSUE #615: Comments from Jason Kalirai (SAC) on preface
- The Preface jumps right into the content of the paper and assumes that the reader is familiar with a lot of terminology ("Project members", "Collaboration", "OpSim", etc..  This is a major publication following the LSST science book and deserves some type of extended "Executive Summary".  This is especially important for the many scientists in the community that haven't been following LSST closely, and will want to read a summary.  This should begin with an inspirational paragraph about the telescope and its survey, and then all of the various terms and communities should be defined explicitly.  The Preface itself is probably OK after such a summary, although I think the word "you" towards the end should probably be replaced with "our community" or similar.

  (Note from Michael: I made a similar comment)

### LINKED ISSUE #616: Comments from Jason Kalirai (SAC) on introduction
- The format of the text in the bullets is not standard.  Some of them list science themes ("dark energy and dark matter") and others are written in a format such as to tackle science themes "Exploring", ""Studying", etc.

### LINKED ISSUE #637: Address Thomas' feedback on Chapter 1 (Introduction)
Sandrine Thomas provided project feedback on Chapter 1 via the LSST Jira [here](https://jira.lsstcorp.org/secure/attachment/29363/WP_Section1Comments_ST.pdf). I can have a go at addressing this. Thanks, Sandrine! :-)

### PR DESCRIPTION
**Pull request by drphilmarshall**
This PR will (eventually) close the following issues:

* [x] Sandrine Thomas' Pub Board review of Chapter 1, closes #637 
* [x] Michael Strauss' Pub Board review of Chapters 1 and 2, closes #611
* [x] Jason Kalirai's SAC review of the Preface, closes #615 
* [x] Jason Kalirai's SAC review of Chapter 1, closes #616 

When I'm done, I'll assign Beth and Zeljko to sign off on these two chapters.

### CODE REVIEW

**Comment by drphilmarshall** [comment]
@michaelstrauss In Chapter 1 I decided to leave in all the "guidelines for contributors" material, partly to show how the WP was made but also to make the arxiv posting a faithful advert for this community project. You also asked, "can a metric be designed even if you don't speak MAF?" The answer to this is "yes" - Peter and Lynne have done a great job at interpreting various science teams explanations and providing initial MAF metrics to them. I addressed the rest of your comments as best as I could, leaving some flags for Zeljko and Beth (below).

@rhiannonlynne While addressing Michael's comment I found that your [DDF white paper link](https://lsstcorp.org/content/whitepapers32012) is broken. So, I linked to the [community thread](https://community.lsst.org/t/deep-drilling-whitepapers/732) instead (which contains the above link). Great if you could fix that or report to (or just edit) the community thread with a replacement. Thanks!

Zeljko, Beth: After addressing Michael's SAC comments on Chapters 1 and 2, I found the following items that I think you need to help out with or sign off on. Great if you can take a look and respond wither by reply here or by tex edit in the `pjm-ch1+2-review-response` branch. Thanks!

In the tex files `introduction.tex` and `cadexp2.tex`, search for the string `@ivezic` to find my comments, and the parts that need checking/editing or agreement. Looks like there's a question for Andy in there as well.

`grep '@ivezic' introduction.tex cadexp2.tex`

> introduction.tex:% @ivezic: please check the following sentences, requested by the Pub Board reviewer.
> introduction.tex:% @ivezic can you please unpack the SOCS acronym and add a sentence about what else this team will do, please? You can refer forward to MAF which is introduced and described in the next section (\ref{sec:intro:evaluation}).
> introduction.tex:% @ivezic: "How will the Project Scientist act? Through the PST, or...?" Can you please suggest some extra detail here, or clean up and ignore?
> introduction.tex:% @ivezic: Please verify or edit the above parenthetical clause, inserted in response to the Pub Board reviewer.
introduction.tex:% @ivezic, @bwillman: Neither the v2.0 nor v3.0 white papers are referred to in the Table, but they are clearly called for in the text. Can you please check these references and make sure you are OK with them? @connolly: should we update the Table to refer to these WP updates?
introduction.tex:% @ivezic: How will all this activity by the SSC and SOCS and

> cadexp2.tex:% @ivezic: Our SAC reviewer suggested including this dec range, and also that we say what the galactic latitude cut is. Can you check the former and add the latter, please? Thanks!

**Comment by drphilmarshall** [comment]
Jason suggested we include an executive summary. I looked back at the Science Book, and found that there we included a short paragraph summarizing the LSST and its motivation in the preface - so to match that I added a similar first paragraph to our preface (and I also unpacked the jargon). Then, since we'll need one for the arxiv posting anyway, I wrote a very short summary and placed it after the preface. Comments/edits welcome! I don't think we need a longer executive summary: the summary that really matters is the one that people read in the arxiv posting(s).

**Comment by ivezic** [comment]
I will probably take care of it today - or by EOB tomorrow at the latest.

File: whitepaper/summary.tex [hunk 1]
```diff
@@ -0,0 +1,46 @@
+\setcounter{chapter}{0}
+\chapter*{Summary}
+\def\chpname{summary}\label{chp:\chpname}
+\addcontentsline{toc}{section}{Summary}
+\markboth{}{}
+
+% Short (c. 250 word) summary for arxiv posting:
+
+\noindent
+The Large Synoptic Survey Telescope is designed to provide a single
```
Comment (drphilmarshall) [inline whitepaper/summary.tex]: @ivezic I omitted to call this out for you to look at in the PR thread or latex comments, sorry - great if you can check and modify my wording for this, the "arxiv abstract". Thanks!

**Comment by ivezic** [comment]
Just push-ed fixes and edits. All good to go, AFAIK.

**Comment by drphilmarshall** [comment]
Excellent - thanks very much, @ivezic! :-)

**Pull request merged**